### DISCUSSION TITLE
Adding firebase plugins to the flet

### DISCUSSION THREAD

**Discussion by zox47** [flet-dev/flet#4902] [Extensions]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

Adding a Firebase plugin to the Flet framework would introduce several powerful features:
https://github.com/firebase/flutterfire

**Comment by ap4499** [comment]
I'm after the same.

Presumably, most devs on Android/iOS are going to want this too.

https://firebase.flutter.dev/docs/overview/
